security.txt

```diff
@@ -1,21 +1,28 @@
 -----BEGIN PGP SIGNED MESSAGE-----
 Hash: SHA256
 
-Contact: mailto:security@keep.network
-Encryption: https://keybase.io/shadowfiend/pgp_keys.asc
-Encryption: https://keybase.io/mhluongo/pgp_keys.asc
+Contact: mailto:security@threshold.network
 Encryption: https://keybase.io/piotrd/pgp_keys.asc
-Acknowledgments: https://keep.network/disclosed-vulnerabilities
+Encryption: https://keybase.io/mhluongo/pgp_keys.asc
+Encryption: https://keybase.io/shadowfiend/pgp_keys.asc
+Acknowledgments: https://threshold.network/disclosed-vulnerabilities
 Preferred-Languages: en, es, fr
-Canonical: https://keep.network/.well-known/security.txt
-Policy: https://github.com/keep-network/keep-core/tree/master/SECURITY.adoc
+Canonical: https://threshold.network/.well-known/security.txt
```

Comment:
The link does not work.

Reply:
Yes because we had no issues reported yet.